apps/system/utils/netcmd_netmon.c

```diff
@@ -100,6 +109,15 @@
 /****************************************************************************
 * Private Functions
 ****************************************************************************/
+/**
+ * Print the command list
+ */
+static void print_help(void)
+{
+	printf("%s", USAGE);
+	return ;
```

Comment:
Is space required?

Reply:
If there is no specific rule for this space, I do not mind to use any of expression.
Modification completed.